kits/cpp/src/lux/unit.cpp

```diff
@@ -53,8 +60,6 @@ namespace lux {
         return std::ceil(unitConfig.DIG_COST * weather.POWER_CONSUMPTION);
     }
 
-    bool Unit::canDig(const Observation &obs) const { return power >= digCost(obs); }
```

Comment:
`canMove` still exists. In the python kit it was removed. Should it be removed here as well?
In general I'm not sure why those utilities are removed. Since the action queue cost lookup can be performed by the unit, I don't see why it shouldn't be able to make the call if it can move or not. Was there a particular reason for this?

Reply:
Yes remove. The reasoning is submitting an action queue it costs power. So canMove etc don't quite make sense since the action queue costa additional power.